### PR TITLE
Fix display of tooltips in script for warnings and errors

### PR DESCRIPTION
**Pull request by jfehrle**
Fix display of tooltips in script for warnings and errors, which were not appearing as expectd.

### CODE REVIEW

File: ide/coqide/coqOps.ml [hunk 1]
```diff
@@ -333,9 +333,7 @@ object(self)
             try Doc.find document mem_marks
             with Not_found -> aux iter#backward_char in
         aux iter in
-      let ss =
-        find_all_tooltips s
-          (iter#offset - (buffer#get_iter_at_mark s.start)#offset) in
+      let ss = find_all_tooltips s iter#offset in
```
Comment (ppedrot) [inline ide/coqide/coqOps.ml:336]: While this is fixing the bug, this is also acknowledging that we're now using absolute offsets.

Comment (jfehrle) [reply ide/coqide/coqOps.ml:336]: I thought we wanted to fix this issue urgently.  This fix doesn't preclude modifying the design of some parts in the near future, but that will take considerably more time to analyze, discuss, code, test and review.  And, importantly, not introduce new bugs.  I will do some analysis on that fairly soon after addressing the urgent parts.

**Review by ppedrot — APPROVED** [review]

**Comment by ppedrot** [comment]
CI failures unrelated. @coqbot merge now

**Comment by JasonGross** [comment]
@jfehrle It seems that this changes the GTK dependencies of CoqIDE?  see #15977.  Probably it shouldn't be included in 8.15?

**Comment by jfehrle** [comment]
@JasonGross I think you're mistaken.  That change is only 4 lines and doesn't use any new calls from GTK. Take a look at it.

**Pull request merged**